plugins/modules/sonic_bgp_neighbors.py

```diff
@@ -87,7 +87,20 @@
           bfd:
             description:
               - Enables or disables BFD.
-            type: bool
+            type: dict
+            suboptions:
+              enabled:
+                description:
+                  - Enables BFD liveliness check for a BGP peer.
+                type: bool
+              check_failure:
+                description:
+                  - Link dataplane status with control plane.
+                type: bool
+              profile:
+                description:
+                  - BFD Profile name format.
```

Comment:
It looks like the word "format" at the end of this description should be deleted.